YDS/Source/Component/YDSBottomBarController.swift

```diff
@@ -75,13 +75,19 @@ open class YDSBottomBarController: UITabBarController {
      각종 프로퍼티를 세팅합니다.
      */
     private func setProperties() {
-        tabBar.tintColor = YDSColor.bottomBarSelected
-        tabBar.unselectedItemTintColor = YDSColor.bottomBarNormal
-        tabBar.backgroundColor = YDSColor.bgElevated
-        tabBar.isTranslucent = false
-        
-        UITabBar.appearance().shadowImage = UIImage()
-        UITabBar.appearance().backgroundImage = UIImage()
+        let appearance = UITabBarAppearance()
+        appearance.configureWithOpaqueBackground()
+        appearance.backgroundColor = YDSColor.bgElevated
+        appearance.shadowColor = .clear
+        appearance.stackedLayoutAppearance.normal.iconColor = YDSColor.bottomBarNormal
+        appearance.stackedLayoutAppearance.normal.titleTextAttributes = [NSAttributedString.Key.foregroundColor : YDSColor.bottomBarNormal]
+        appearance.stackedLayoutAppearance.selected.iconColor = YDSColor.bottomBarSelected
+        appearance.stackedLayoutAppearance.selected.titleTextAttributes = [NSAttributedString.Key.foregroundColor : YDSColor.bottomBarSelected]
```

Comment:
UITabBarItemAppearance을 써보는건 어떤가요

맨 아래 compactInline이랑 inline은 각각 작은화면 가로모드랑 아이패드에서의 Appearance인데, 필요없으면 빼도 됩니다.

```suggestion
        let tabBarItemAppearance = UITabBarItemAppearance(style: .stacked)
        tabBarItemAppearance.normal.iconColor = YDSColor.bottomBarNormal
        tabBarItemAppearance.normal.titleTextAttributes = [.foregroundColor : YDSColor.bottomBarNormal]
        tabBarItemAppearance.selected.iconColor = YDSColor.bottomBarSelected
        tabBarItemAppearance.selected.titleTextAttributes = [.foregroundColor : YDSColor.bottomBarSelected]
        appearance.stackedLayoutAppearance = tabBarItemAppearance
        appearance.compactInlineLayoutAppearance = tabBarItemAppearance
        appearance.inlineLayoutAppearance = tabBarItemAppearance
```

Reply:
이것도 있는 줄은 몰랐어요

Reply:
8fe6bccfaffbc7d1286d83ed7c5ec384911c9c65

폰만 생각하느라 패드는 미처 생각하지 못했네요 짚어주셔서 감사합니다!